app/src/app.ts

```diff
@@ -1,39 +1,47 @@
-import * as express from "express";
-import * as bodyParser from "body-parser";
-import { Routes } from "./routes/api";
-import * as mongoose from "mongoose";
+import express from 'express';
+import bodyParser from 'body-parser';
+import { Routes } from './routes/api';
+import mongoose from 'mongoose';
 
+/**
+ * Main app to configure express server with mongo connection
+ */
 class App {
-
     public app: express.Application = express();
     public routes: Routes = new Routes();
     // TODO secure mongodb access with username and password
     public mongoUrl: string = 'mongodb://mongo:27017/expressmongo';
 
+    /**
+     * Run several other configuration methods and mount routes
+     */
     constructor() {
-        this.config();
-        this.mongoSetup();
-        this.routes.routes(this.app);
+      this.config();
+      this.mongoSetup();
+      this.routes.routes(this.app);
     }
 
+    /**
+     * Configuration for express server
+     */
     private config(): void{
```

Comment:
eslint should fix such moments. 
Maybe this rule can help https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/type-annotation-spacing.md

Reply:
@neSpecc could not find any rule for that. i fixed it manually for now